usercp.php

```diff
@@ -3348,6 +3351,63 @@
 	redirect("usercp.php?action=attachments", $lang->attachments_deleted);
 }
 
+
+if($mybb->input['action'] == "warninglog")
+{
```

Comment:
Since this page will not be shown to users based on their warning level (=0), perhaps block it as well based on the same logic.